client.go

```diff
@@ -289,6 +289,15 @@ type Config struct {
 	// setting of Postgres `search_path`.
 	Schema string
 
+	// SkipJobKindValidation causes the job kind format validation check to be
+	// skipped. This is available as an interim stopgap for users that have
+	// invalid job kind names, but would rather disable the check rather than
+	// fix them immediately.
+	//
+	// Deprecated: This option will be removed in a future versions so that job
+	// kinds will always have to have a valid format.
+	SkipJobKindValidation bool
```

Comment:
So this is more of an opt-out than an opt-in. Maybe it will be fine though? Really hard to predict how people are using this IRL.

Reply:
Yeah, I'd originally started with an opt-in, but two things occurred to me while doing that:

* Adding/removing one of these configs is a breaking change, and with an opt-in, we'd probably _also_ want an opt-out later, so: (1) add opt-in, (2) make validation required + add opt-out, (3) deprecate opt-in option, (4) deprecate opt-out option. So that's two breaking changes instead of one.
* Very likely, few people would notice the new opt-in option and actually use it.

So I'm pretty sure the net effect would be more churn and not much pain saved from the extra opt-in, so just decided to jump to an opt-out instead.